src/software/ai/navigator/path_planner/theta_star_path_planner.cpp

```diff
@@ -470,21 +479,22 @@ bool ThetaStarPathPlanner::isPointValidAndFreeOfObstacles(Point p)
     return true;
 }
 
-bool ThetaStarPathPlanner::isPointValid(Point p)
+bool ThetaStarPathPlanner::isPointValid(const Point &p) const
```

Comment:
Not blocking, but as a thought, `isPointWithinNavigableRegion` might be a bit more descriptive here.

Reply:
changed